### DISCUSSION TITLE
rest Service creation

### DISCUSSION THREAD

**Discussion by zyzgreater** [apache/camel-karavan#1221] [Q&A] [answered]
Hello, I want to create an http service and plan to use camel's rest mode, but there is a problem, may I ask what is wrong with my configuration? thank you.
![image](https://github.com/apache/camel-karavan/assets/98529866/add20b20-ad14-43f1-9e1d-8810cf66912f)
![image](https://github.com/apache/camel-karavan/assets/98529866/dc1b8190-4271-4ca6-a479-8b613af848e5)
![image](https://github.com/apache/camel-karavan/assets/98529866/277cd620-3502-40be-9c0e-cd819835754a)
![image](https://github.com/apache/camel-karavan/assets/98529866/f5e45bf1-81ca-42fb-b489-093ccb4f8a13)

**Comment by mgubaidullin** [comment]
You may ask what is wrong with your configuration, but to get answer you need to share your configuration. As YAML file.

**Reply by zyzgreater** [reply]
This is the generated yaml
![image](https://github.com/apache/camel-karavan/assets/98529866/2a23149c-a635-4a3d-9d81-9328f240aed1)
![image](https://github.com/apache/camel-karavan/assets/98529866/18994178-4e5d-491f-8a49-9c9550bc4dd2)
![image](https://github.com/apache/camel-karavan/assets/98529866/0f1ddbde-daca-453d-8fac-d7be46bf6e3e)

**Reply by zyzgreater** [reply]
![image](https://github.com/apache/camel-karavan/assets/98529866/4fc963d9-c5f8-4032-9619-dd4713cf36dc)
![image](https://github.com/apache/camel-karavan/assets/98529866/72031bf3-3c9c-4ec6-a275-2103e1adb36e)

**Comment by zyzgreater** [comment]
@mgubaidullin

**Comment by mgubaidullin** [comment, marked as answer]
Simple rest service example

<img width="507" alt="Screenshot 2024-04-12 at 9 20 32 AM" src="https://github.com/apache/camel-karavan/assets/1379213/ba1a9a2d-02dd-4876-9d29-b0513b97a6df">

**Reply by zyzgreater** [reply]
![image](https://github.com/apache/camel-karavan/assets/98529866/003d0c67-18c1-4718-94b0-38360135d8c5)
![image](https://github.com/apache/camel-karavan/assets/98529866/983fa199-3b93-4bff-8922-7d5beb7a057b)
Excuse me, I am using the example you provided, after running, I reported an error, where should I set this configuration, thank you

**Reply by zyzgreater** [reply]
![image](https://github.com/apache/camel-karavan/assets/98529866/3dd879df-6e48-4cfe-a773-e7e25bac5d7b)
![image](https://github.com/apache/camel-karavan/assets/98529866/aee86255-4f48-418a-8915-50b85c3e9d68)
![image](https://github.com/apache/camel-karavan/assets/98529866/ed9ecf52-10a4-4fe8-9307-b535197725dc)
I added this configuration and it worked, the health check interface worked, but the custom get request returned an error

**Reply by zyzgreater** [reply]
![image](https://github.com/apache/camel-karavan/assets/98529866/def975b7-95d1-4b46-a286-d5d0a7ecf758)
I added another div and it worked, thanks for your patient help

**Reply by davsclaus** [reply]
okay so this works for you now ?

**Reply by zyzgreater** [reply]
Yes, the desired function has been completed so far, and I will continue to explore other functions, thank you
